pypicloud/cache/base.py

```diff
@@ -6,6 +6,7 @@
 from pyramid.settings import asbool
 
 import posixpath
+from flywheel.fields.types import UTC
```

Comment:
Either is fine. I usually try to keep the dependencies to a minimum, but I realize that that's because all my other projects are libraries. For pypicloud it really doesn't matter.